sms-checker/README.md

```diff
@@ -0,0 +1,8 @@
+# SMS-Checker Pipeline
+This folder contains the code for the SMS-checker application. This application can be triggered to query the database for any subscribers that are due an alert for a celestial body or aurora, this is done using the `psycopg2` library. These subscriber details are then used to send emails and SMS messages with the appropriate alerts using `boto3`.
```

Comment:
Good discussion around the later functionality around what groups of subscribers should be notified about visible celestial bodies and/or events are near them. We need to keep this presently in mind so we build our earlier functionality around this key feature.